pkgs/development/python-modules/demes/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ruamel-yaml
+, attrs
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "demes";
+  version = "0.2.3";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9e613b65b475dba277bca751de1f37a89fd5eb4d856ba27733a2099c8a823617";
```

```suggestion
    hash = "9e613b65b475dba277bca751de1f37a89fd5eb4d856ba27733a2099c8a823617";
```
Has the same effect but is nicer to read since it's obvious that its sha256.